spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -54,10 +55,18 @@
 
       it 'shows failure on intentionally bad values' do
         post :create, first_name: 'Bad', ssn: '6666'
+        post :update, id: 1, ccn: '1234123412341234'
 
         expect(response).to redirect_to(idv_sessions_path)
         expect(flash[:error]).to eq t('idv.titles.fail')
       end
+
+      it 'shows failure on intentionally bad values' do
+        post :create, first_name: 'Bad', ssn: '6666'
+        post :update, id: 1, ccn: '1234123412341234'
```

Comment:
Note we probably don't want to collect more than the last 8 digits -- i.e. explicitly disallow the full CC, to re-enforce that we don't need the whole CC# and nothing is being charged.